toolchains/syno-purley-6.2.2/Makefile

```diff
@@ -16,7 +16,13 @@ TC_TARGET = x86_64-pc-linux-gnu
 TC_CFLAGS = -I$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/usr/include
 TC_CPPFLAGS = -I$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/usr/include
 TC_CXXFLAGS = -I$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/usr/include
-TC_LIBDIR = $(TC_BASE_DIR)/sys-root/lib
-TC_LDFLAGS = -L$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_LIBDIR)
+TC_LDFLAGS = -L$(WORK_DIR)/$(TC_BASE_DIR)/$(TC_BASE_DIR)/sys-root/lib
```

Comment:
These lines are identical in most toolchains... I would set these values as defaults in `spksrc.tc.mk` if not already set in arch specific toolchain (like armv7 toolchains where additional options are required)

Reply:
I already started to devine a TC_LIBDIR and to adjust the FIX_TARGET in spksrc.tc.mk but moved it to a separate local work. I will try to merge that into this PR.